full/Dockerfile

```diff
@@ -119,7 +67,7 @@ ENV PATH="$PATH:/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin/"
     MANPATH="$MANPATH:/home/linuxbrew/.linuxbrew/share/man" \
     INFOPATH="$INFOPATH:/home/linuxbrew/.linuxbrew/share/info" \
     HOMEBREW_NO_AUTO_UPDATE=1
-RUN sudo apt-get remove -y cmake \
+RUN sudo install-packages cmake \
     && brew install cmake
```

Comment:
Whoops. This line actually intended to _remove_ the `cmake` installed by `apt-get` (so that it can be replaced by the one installed with `brew`).

But maybe this isn't even needed anymore? We should check which version is installed by `apt-get` and which by `brew`. If `apt-get` already gives the latest stable, we can remove these two lines.

Reply:
Ubuntu package version is `3.16.3` and brew `3.19.5`

Reply:
fixed